rules/list-item.js

```diff
@@ -54,7 +54,7 @@ module.exports = rule('remark-lint:awesome-list-item', (ast, file) => {
 	const toc = find(ast, node => (
 		node.type === 'heading' &&
 		node.depth === 2 &&
-		toString(node) === 'Contents'
+		toString(node).replace(/<!--.*?-->/g, '').trim() === 'Contents'
```

Comment:
I think you can use the `includeHtml` for `toString` instead.

Reply:
Unfortuntaely, that didn't work:

> Note that the toString() function does accept an option called includeHtml, but unfortunately that did not help in this case. So this regex seemed like the next best alternative.

Reply:
Sorry, I missed that.